server/server_test.go

```diff
@@ -1,6 +1,7 @@
 // Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
+//go:build !integration
```

Comment:
Do we really need to add these build tags to every single non-integration test file now?

In beats it's done in a reversed way: https://github.com/elastic/beats/blob/678db731918abba48169f8a2147e6f1df66bf77e/libbeat/outputs/kafka/kafka_integration_test.go#L18-L19

Reply:
Actually this is like beats.  Integrations tests have `//go:build integration` and unit tests have `//go:build !integration`

For example, this is a unit test in the same directory as your example.
https://github.com/elastic/beats/blob/277776a46152b45bc4ccb516b0e453650983f10d/libbeat/outputs/kafka/partition_test.go#L18-L19

I'm not fond of this approach, and I'm very open to suggestions.

Reply:
Yeah I'm not fond of the build tags either, but it is the most reliable way to make integration tests opt-in that I know of. The other approach I know of is remembering to run unit tests with `go test -short` and checking `if !testing.Short()` in all the integration tests.